README.md

```diff
@@ -127,28 +129,18 @@ fetcher.use(logger)
 
 ### Server Side Usage
 
-This library can be used server side with [node-fetch](https://www.npmjs.com/package/node-fetch)
-
-Node CommonJS setup
+This library can be used server side if you have a compatible fetch lib (e.g. [cross-fetch](https://www.npmjs.com/package/cross-fetch)):
+```sh
+npm install cross-fetch
+```
+You will then need to inject `fetch` into your instance:
 ```ts
-// install node-fetch v2
-npm install node-fetch@2
-npm install @types/node-fetch@2
-
-// fetch-polyfill.ts
-import fetch, { Headers, Request, Response } from 'node-fetch'
-
-if (!globalThis.fetch) {
-    globalThis.fetch = fetch as any
-    globalThis.Headers = Headers as any
-    globalThis.Request = Request as any
-    globalThis.Response = Response as any
-}
-
-// index.ts
-import './fetch-polyfill'
+import fetch from 'cross-fetch'
+const fetcher = Fetcher.for<paths>(fetch)
```

Comment:
Is this the correct setup still? I don't see any code changes to using
```
const fetcher = Fetcher.for<paths>();
fetcher.configure({ fetchProvider: { fetch, headers });
```

Reply:
thanks, i wrote that before. will change.